src/vs/editor/browser/controller/textAreaHandler.ts

```diff
@@ -575,7 +575,11 @@ export class TextAreaHandler extends ViewPart {
 	public override onCursorStateChanged(e: viewEvents.ViewCursorStateChangedEvent): boolean {
 		this._selections = e.selections.slice(0);
 		this._modelSelections = e.modelSelections.slice(0);
-		this._textAreaInput.writeScreenReaderContent('selection changed');
+		if (this._accessibilitySupport === AccessibilitySupport.Enabled) {
```

Comment:
We should be more defensive about this. On the web, accessibility support is most of the times `Unknown` and only on the desktop we get to know if it's on or not (`Enabled` or `Disabled`). In code, that means we should use `if (this._accessibilitySupport !== AccessibilitySupport.Disabled)`.